src/compiler/failsafe.cc

```diff
@@ -389,17 +367,40 @@ class FailSafeCompiler : public Compiler {
   }
 
  private:
-  CompilerParam param;
+  CompilerParam param_;
   int num_feature_;
   unsigned int num_class_;
   std::string pred_tranform_func_;
   std::unordered_map<std::string, CompiledModel::FileEntry> files_;
 };
 
-TREELITE_REGISTER_COMPILER(FailSafeCompiler, "failsafe")
-.describe("Simple compiler to express trees as a tight for-loop")
-.set_body([](const CompilerParam& param) -> Compiler* {
-    return new FailSafeCompiler(param);
-  });
+FailSafeCompiler::FailSafeCompiler(const CompilerParam& param)
+    : pimpl_(std::make_unique<FailSafeCompilerImpl>(param)) {
+  if (param.verbose > 0) {
+    LOG(INFO) << "Using FailSafeCompiler";
```

Comment:
Is there a log warning macro?

Reply:
LOG(WARN) is just an alias of LOG(INFO). Treelite doesn't let you set global verbosity level like XGBoost.

Reply:
Got it.